integrate/events/tutorial/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Events Tutorial
+permalink: /integrate/events/tutorial/
+---
+
+* TOC
+{:toc}
+
+
+In this tutorial, we will setup a receiver for the [`issues`](https://docs.github.com/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#issues) event from Github Webhooks. This will allow you to monitor and analyse activity relating to issues on any of your Github repositories.
+
+You will need your project's master token, and a Github repository in which you have the `Admin` role.
+
+To start ingesting events, you must first create a receiver. Send the following payload to the `https://buffer.keboola.com/v1/receivers` endpoint:
+```
+{
+  "name": "Github Issues",
+  "exports": [
+    {
+      "name": "Events",
+      "conditions": {
+        "time": "2m"
+      },
```

Comment:
Toto by som odstranil, vysvetlit, ze sa to importuje kazdu `1m` a dat link na podrobnejsie vysvetlenie conditions.